core/src/main/java/com/rim/IceField/Eskimo.java

```diff
@@ -13,6 +15,11 @@ public Eskimo() {
 
     }
 
+    @Override
```

Comment:
run? could you please a more meaningful name? It's not very clear what this method does